spec/rubocop/target_ruby_spec.rb

```diff
@@ -188,10 +188,25 @@
 
       context 'when file reads the required_ruby_version from another file' do
         it 'uses the default target ruby version' do
-          content = <<~HEREDOC
+          content = <<~'HEREDOC'
+            Gem::Specification.new do |s|
+              s.name = 'test'
+              s.required_ruby_version = Gem::Requirement.new(">= #{File.read('.ruby-version').rstrip}")
+              s.licenses = ['MIT']
+            end
+          HEREDOC
+
+          create_file(gemspec_file_path, content)
+          expect(target_ruby.version).to eq default_version
+        end
+      end
+
+      context 'when file reads the required_ruby_version from another file in an array' do
+        it 'uses the default target ruby version' do
```

Comment:
This is the new spec that I am adding for this change. Unfortunately (due to how it looks similar to the spec changed above), GitHub is actually not even including the whole new spec in its rendering of this diff. You'll have to click the down arrow to see the last few lines of this new spec, which GitHub is rendering as if they are preexisting lines.